examples/step-5/step-5.cc

```diff
@@ -113,6 +113,8 @@ double coefficient (const Point<dim> &p)
   else
     return 1;
 }
+// @note Mapping of this representation of the coefficient to the mesh cell
+// is non-trivial, see Results section of step-6 for more discussion.
```

Comment:
Good! I will try to combine few words on material_id  and on non-uniform coefficient distribution into the wording from @bangerth , I hope I will find some time on tomorrow to do it.